server/src/main/java/io/crate/planner/operators/Union.java

```diff
@@ -134,8 +134,8 @@ public ExecutionPlan build(DependencyCarrier executor,
             left,
             right,
             mergePhase,
-            limit,
-            offset,
+            LimitAndOffset.NO_LIMIT,
+            LimitAndOffset.NO_OFFSET,
```

Comment:
This was wrong and resulted in execution plans that applied a limit before a order by.
The `MoveOrderBeneathUnion` optimization rule helped mask this bug. So for users there was no incorrect behavior, assuming the rule is active.

Reply:
👍